src/test/java/neculaclaudiu/tests/seleniumpart1/TestAlDoileaSite.java

```diff
@@ -0,0 +1,127 @@
+package neculaclaudiu.tests.seleniumpart1;
+
+import neculaclaudiu.driver.ChromeDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+
+import java.util.List;
+
+public class TestAlDoileaSite {
+    public static void main(String[] args) {
+        TextBoxCase();
+        CheckBoxCase();
+        RadioButtonCase();
+        WebTablesCase();
+        ButtonsCase();
+
+    }
+    public static void TextBoxCase(){
```

Comment:
Name of the methods are still UpperCase and not lowerCase. Please make this change too. Thanks!